### ISSUE TITLE
MSI GeForce GTX 1080 Ti SEA HAWK EK X (Temperature Based issue)

### ISSUE THREAD

**Issue by sebby47** [Vipeax/MSI-LED-Tool#41]
First off I'd just like to say thank you so much for this software, we all know how annoying the msi software is so having this is incredibly comforting. 

My issue itself revolves around the temperature based setting, it seems to be working fine in that it registers when the temp gets high and low. However, there is no progressive change in between it simply moves straight from green to yellow then red. My mistake if this is normal but in the readme it states it should be progressive and contain a few colors in the spectrum before it hits red. 

Kind regards, Sebastian

**Comment by MinDBreaK** [comment]
If you want, i will update my tool for nVidia. Color should be patched to integrate a better color management. 

It work well on my config with AMD, but I need a tester on nVidia

**Comment by sebby47** [comment]
Hey there, thanks for the help, but I'm not sure I follow. Are you saying you have your own software that should work? Can Help at all?

**Comment by MinDBreaK** [comment]
Yes my version should work once I finished the implementation of nVidia, but I don't own nVidia card so I can't test it. 

But you could be the beta tester ;)

As I recently cleaned my computer, I just have to reinstall Visual Studio. But it should be done for this weekend (if I remember to do it tonight).

**Comment by sebby47** [comment]
I'd be more than happy to help out, just let me know what you need me to do! Thanks

**Comment by MinDBreaK** [comment]
Hello !

I'm back, sorry for the delay 

I currently pushed a new version on my own repo here : https://github.com/MinDBreaK/MSI-LED-Control

You currenly need to compile it with Visual Studio. 
If needed, if can provide an executable

**Comment by sebby47** [comment]
I'm not sure how to do that! sorry

**Comment by MinDBreaK** [comment]
Then I will provide you a zip. You will need to download and extract it wherever you want. Then launch it and enjoy (and report also)

Your antivirus might be triggered. That's because it's not correcly signed (but i don't know how to do it)

The file will be on my repo, so be sure to check it out

**Comment by sebby47** [comment]
I'll let you know how it goes, thanks buddy!

**Comment by sebby47** [comment]
Hey Antoine,
I cant seem to get it to work, is there something else I should know or should let you know?

**Comment by MinDBreaK** [comment]
So you can't start it ? Then I'm going to add some debug components. I'll come back to you 

Be sure that you have the last version of .NET Framework V4
